src/vm/domainfile.cpp

```diff
@@ -3203,6 +3203,9 @@ void GetTimeStampsForNativeImage(CORCOMPILE_VERSION_INFO * pNativeVersionInfo)
 #endif // FEATURE_CORECLR
 }
 
+static constexpr uint32_t CMOVFlags = 0x00008001;
+static constexpr uint32_t SSE2Flags = 0x04000000;
```

Comment:
Probably for the best... https://github.com/dotnet/coreclr/issues/6642 seems there is compiler pain with it